src/components/ScenarioParameters/components/ScenarioParametersTabs/ScenarioParameterInput.js

```diff
@@ -35,9 +35,10 @@ const ScenarioParameterInput = ({ parameterData, context }) => {
   return (
     <Controller
       name={parameterData.id}
-      render={({ field }) => {
+      render={({ field, fieldState }) => {
         const { value: parameterValue, onChange: setRhfValue } = field;
-
+        const { isDirty } = fieldState;
+        const context = { ...tabContext, isDirty };
```

Comment:
Same remarq than @csm-thu  , i think is better to add isDirty as a optional prop to all generic input, use it in UI for let basic components apply their particular style.